index.html

```diff
@@ -0,0 +1,177 @@
+    integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;0,900;1,400;1,700;1,900&display=swap"
+    rel="stylesheet">
+  <link rel="stylesheet" href="./css/styles.css">
+  <link rel="stylesheet" href="./css/desktop.css">
+
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
+    integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
+    crossorigin="anonymous"></script>
+  <script defer src="./js/script.js"></script>
+</head>
+
+<body>
+  <header>
+    <nav class="mobile-nav">
+      <img src="./images/mobile-menu-icon.png" alt="Mobile menu icon" class="mobile-menu-icon">
+      <a href="./index.html"><img src="./images/tech-splendor.png" alt="Project logo" class="project-logo"></a>
+
+      <div class="mobile-menu">
+        <img src="./images/close-menu.png" alt="Close mobile menu icon" class="close-menu-icon">
+        <ul class="menu">
+          <li><a href="./about.html">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+          <li><a href="#">TS Campaign</a></li>
+        </ul>
+      </div>
+    </nav>
+
+    <nav class="desktop-nav">
+      <ul class="socials">
+        <li><a href=""><img src="./images/socials/facebook.svg" alt="Facebook icon"></a></li>
+        <li><a href=""><img src="./images/socials/twitter.svg" alt="Twitter icon"></a></li>
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Page</a></li>
+        <li><a href="#">Logout</a></li>
+      </ul>
+
+      <div>
+        <a href="./index.html">
+          <img src="./images/tech-splendor.png" alt="Project Logo" class="desktop-project-logo">
+        </a>
+        <ul class="menu desktop-menu">
+          <li><a href="./about.html">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+          <li><a href="#">TS Campaign</a></li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+
+  <section class="hero">
+    <q>Hello! Tech World!</q>
+    <h1>TECHSPLENDOR SYMPOSIUM 2023</h1>
+    <p class="intro">
+      TechSplendor, the pinnacle of technological excellence! Prepare to immerse yourself in an extraordinary tech
+      journey like never before.
+    </p>
+    <p class="schedule">2023.10.16(MON) ~ 23rd(FRI)</p>
+    <p class="venue">
+      @ Mayor's Garden of Lira, Coronation Park and more
+    </p>
+  </section>
+
+  <main>
+    <section class="main-program">
```

Comment:
- [ ] [REQUIRED] Good job so far! Please pay attention to size of `horizontal line` and give required `hover effect` like a [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏🏼  

![image](https://github.com/dd-obua/Mic-Portal-Captone/assets/63915024/ca00ff5b-2b65-40ae-a12f-ed0cabe7e8ad)